cmd/ctlog/createctconfig/main.go

```diff
@@ -57,6 +60,7 @@ var (
 	ctlogPrefix        = flag.String("log-prefix", "sigstorescaffolding", "Prefix to append to the url. This is basically the name of the log.")
 	fulcioURL          = flag.String("fulcio-url", "http://fulcio.fulcio-system.svc", "Where to fetch the fulcio Root CA from")
 	trillianServerAddr = flag.String("trillian-server", "log-server.trillian-system.svc:80", "Address of the gRPC Trillian Admin Server (host:port)")
+	keyType            = flag.String("keytype", "ecdsa", "Which private key to generate [rsa,ecdsa")
```

Comment:
We could also support ed25519 at some point if we wanted to, but i know that sigstore-python doesn't support it currently, and i'm not sure if other clients do. When writing the architecture docs, we'll want to decide whether to only support ecdsa and rsa, or mandate that clients support ed25519 too. i'm not incredibly opinionated here - If all languages had as nice of a key abstraction as golang, I'd say support all three, but it's probably unnecessary

Reply:
Ok, fine as a follow up?

Reply:
Yep, I think leave this as a TODO and we can implement this after clients add support